docs/hw/memory/g80-surface.rst

```diff
@@ -99,22 +99,22 @@ Or, alternatively, the address of byte (x,y) is::
     address + pitch * y + x
 
 
-Tiled surfaces
-==============
+Blocklinear surfaces
+====================
 
-A tiled surface is a 3d array of elements, stored in memory in units called
-"tiles". There are two levels of tiling. The lower-level tile is called
-a "roptile" and has a fixed size. This size is 64 bytes × 4 × 1 on G80:GF100
-cards, 64 bytes × 8 × 1 for GF100+ cards. The higher-level tile is called
-a bigtile, and is of variable size between 1×1×1 and 32×32×32 roptiles.
+A blocklinear surface is a 3d array of elements, stored in memory in units
+called "blocks". There are two levels of tiling. The lower-level
+block is called a "gob" and has a fixed size. This size is 64 bytes × 4 × 1
+on G80:GF100 cards, 64 bytes × 8 × 1 for GF100+ cards. The higher-level block is
+called a block, and is of variable size between 1×1×1 and 32×32×32 gob.
```

Comment:
But then I should change that before as well, so in "lower-level block" and "stored in memory in units called \"blocks\"", which will end up weird as well...